gv.c

```diff
@@ -1548,30 +1548,28 @@ S_require_tie_mod(pTHX_ GV *gv, const char varname, const char * name,
     S_require_tie_mod(aTHX_ gv, varname, STR_WITH_LEN(name), flags)
 
 /*
-=for apidoc gv_stashpv
+=for apidoc      gv_stashpv
+=for apidoc_item gv_stashpvn
+=for apidoc_item gv_stashpvs
+=for apidoc_item gv_stashsv
 
-Returns a pointer to the stash for a specified package.  Uses C<strlen> to
-determine the length of C<name>, then calls C<gv_stashpvn()>.
+Note C<gv_stashsv> is strongly preferred for performance reasons.
```

Comment:
This claim re performance appears to have been first introduced here:
```
commit 808724c8e7a94623556f18e681fba068b52291a5 (HEAD)
Author:     syber <syber@crazypanda.ru>
AuthorDate: Mon Sep 1 23:47:54 2014 +0200
Commit:     Yves Orton <demerphq@gmail.com>
CommitDate: Tue Sep 2 00:32:54 2014 +0200
```
Do we still believe it to be true?  If so, do we have benchmarking to back the claim up?

Reply:
I took that as true without investigation

Reply:
During compilation literal stash names, like in `Foo->method()` the stash name is stored as a shared SV, which has a slot to store the hash of the string, which saves some time in doing the lookup.

I don't see what else it could be.